src/backend/base/langflow/components/model_specs/GoogleGenerativeAISpecs.py

```diff
@@ -46,7 +46,7 @@ def build_config(self):
             "model": {
                 "display_name": "Model",
                 "info": "The name of the model to use. Supported examples: gemini-pro",
-                "options": ["gemini-pro", "gemini-pro-vision"],
+                "options": ["gemini-1.5-pro", "gemini-1.5-flash", "gemini-1.0-pro", "gemini-1.0pro-vision"],
```

Comment:
Hey @h-arnold 
Thanks for the PR! 

You got a typo here: "gemini-1.0pro-vision"

Reply:
Apologies and well spotted! I've fixed it now so hopefully that it's good to go. :)